package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "deploy": "gh-pages -d dist",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "server": "node server.js",
+    "build": "webpack"
```

Comment:
looks like you have 2 scripts that do the same, "server" and "start", but with different tools